data_analysis/notebooks/businesses.ipynb

```diff
@@ -3,9 +3,20 @@
   {
```

Comment:
Interesting, I was unaware of f strings thank you for the suggestion, I think it is better. I defaulted to 6 purely because it is what they use for the Denver Metro region, I could have left it blank or as a different region. but didn't want to initially scrape all they had.